### PR TITLE
Move code from IDR-metadata PR

### PR DESCRIPTION
**Pull request by dominikl**
Moves all the code from https://github.com/IDR/idr-metadata/pull/297 into this separate 'idr0040-aymoz-singlecell' repository.

I also added all my Java 'scripts' I used to prepare the `annotation.csv` and `filePaths.tsv`, and various other scripts (e. g. dealing with the rendering settings) to the `scripts` directory and a brief `scripts/workflow.txt` to explain what I actually did in order to prepare the files.

### CODE REVIEW

File: scripts/rendering/copy_rend_settings.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# 
+# The rendering settings are stored as display_and_comments.txt within each
+# image directory. This script just collects (copies) them and renames 
+# them as xyz_mm_rend.json (where xyz is the image directory)
+
+for i in /uod/idr/filesets/idr0040-aymoz-singlecell/20180215/**/display_and_comments.txt
+do
+	IFS='/' read -ra arr <<< "$i"
+	cp $i /home/dlindner/${arr[6]}_mm_rend.json
```
Comment (sbesson) [inline scripts/rendering/copy_rend_settings.sh]: This probably needs to be update to use relative paths ?

File: scripts/annotations/CSVMerger.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+ * Merges several CSV files into a single CSV file. First line of the CSV files
+ * must be a header line! Ignores entries starting with a '#' (also whole
+ * columns if their header starts with a '#'!)
+ */
+public class CSVMerger {
+
+    // BEGIN Parameters
+    
+    static final String[] files = {
+            "/Users/dlindner/idr0040/idr0000-experimentB-assays_DoseResponsesSupFigs.csv",
```
Comment (sbesson) [inline scripts/annotations/CSVMerger.java]: Should these files be moved under revision control and/or the paths adjusted to point to `/uod/idr/filesets`?

**Review by sbesson — APPROVED** [review]
Overall the annotation aspect has been extensively discussed in https://github.com/IDR/idr-metadata/pull/297 and I would be inclined to get this merged asap as it captures the state of the metadata files used for the import into the testing server.

A coupled of comments on the scripts used for generating the annotation files (:+1: for committing them). Overall there might be a review of all these scripts across repos so that we can start refactor common functionalities upstream.

**Comment by dominikl** [comment]
That's true, changed the paths to relative paths throughout, looks better. And added reference as comment to original location under `/uod/idr/filesets` where appropriate.

**Pull request merged**